### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by thomasgaudelet**

### CODE REVIEW

File: pyrelational/models/mcdropout_model.py [hunk 1]
```diff
@@ -42,6 +42,7 @@ def __call__(self, loader: DataLoader) -> torch.Tensor:
             raise ValueError("No current model, call 'train(train_loader, valid_loader)' to train the model first")
         predictions = []
         model = self.current_model
```
Comment (paulmorio) [inline pyrelational/models/mcdropout_model.py:44]: As a generic question does model.eval() capture other things than gradients that should be in "evaluation" mode not caught by with torch.no_grad()?

**Review by paulmorio — APPROVED** [review]
Thanks for updating the documentation both on the website as well as on the DataManager. LGTM

File: pyrelational/models/mcdropout_model.py (hunk 1 above)
Comment (thomasgaudelet) [reply pyrelational/models/mcdropout_model.py:44]: I think it handles dropout and batchnorms (and maybe other modules that behave differently during training and eval)
